libnemo-private/nemo-global-preferences.h

```diff
@@ -214,7 +214,9 @@ typedef enum
 #define NEMO_PREFERENCES_LOCKDOWN_COMMAND_LINE         "disable-command-line"
 
 /* Desktop background */
-#define NEMO_PREFERENCES_SHOW_DESKTOP		   "show-desktop-icons"
+#define NEMO_PREFERENCES_DESKTOP_LAYOUT "desktop-layout"
+#define NEMO_PREFERENCES_SHOW_ORPHANED_DESKTOP_ICONS "show-orphaned-desktop-icons"
+#define NEMO_PREFERENCES_SHOW_DESKTOP   "show-desktop-icons"    /* DEPRECATED */
```

Comment:
This feature is currently not only deprecated, but removed. 
This removes also the flexibility of using other file managers than Nemo with Mint cinnamon and it removes also the option to use Nemo in other desktop environments. 

Something similar described here:
http://askubuntu.com/questions/301089/no-desktop-icons-after-uninstalling-nemo
http://www.noobslab.com/2014/02/set-nemo-as-default-file-manager-in.html

How will it be possible to allow these combinations without this "NEMO_PREFERENCES_SHOW_DESKTOP" in future? 

I think it is not a big deal, to not start the new desktop manager conditional by this flag, or do I miss something?